### PR TITLE
[DNX SAI update] [master] bump DNX SAI version to SAI 10.1.13.1

### PR DESCRIPTION
**Pull request by wenyiz2021**
BRCM SAI ver: [10.1.13.0], OCP SAI ver: [1.13.2], SDK ver: [sdk-6.5.29]

##### Work item tracking
- Microsoft ADO **(number only)**: 27535691

#### How to verify it
SAI bumped up:
```
chassis-linecard$bcmcmd "bcmsai ver"
BRCM SAI ver: [10.1.13.0], OCP SAI ver: [1.13.2], SDK ver: [sdk-6.5.29]
```

dockers come up fine on linecard.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: platform/broadcom/sai.mk [hunk 1]
```diff
@@ -12,7 +12,7 @@ $(eval $(call add_derived_package,$(BRCM_XGS_SAI),$(BRCM_XGS_SAI_DEV)))
 $(BRCM_XGS_SAI_DEV)_URL = "$(LIBSAIBCM_XGS_URL_PREFIX)/$(BRCM_XGS_SAI_DEV)"
 
 # SAI module for DNX Asic family
-BRCM_DNX_SAI = libsaibcm_dnx_$(LIBSAIBCM_DNX_VERSION)_amd64.deb
+BRCM_DNX_SAI = libsaibcm_$(LIBSAIBCM_DNX_VERSION)_amd64.deb
```
Comment (arlakshm) [inline platform/broadcom/sai.mk:15]: the name changed from `libsaibcm_dnx` to `libsaibcm` ?

Comment (wenyiz2021) [reply platform/broadcom/sai.mk:15]: hi @arlakshm yes, this name is changed to match our azure storage name, for SAI 10.x uploaded to azure storage, the DNX sai names changed to this. it is still under ./dnx directory to differentiate with XGS

**Comment by liushilongbuaa** [comment]
> hi @liushilongbuaa , please help review this PR on the change to `$(BRCM_DNX_SAI)_URL`

It's not code conflict. You upgrade the version from 7 to 10.

**Comment by wenyiz2021** [comment]
> > hi @liushilongbuaa , please help review this PR on the change to `$(BRCM_DNX_SAI)_URL`
> 
> It's not code conflict. You upgrade the version from 7 to 10. I don't know if it is by design. I can help ms_conflict pass.

thanks, can you help to pass ms conflict? we want to upgrade SAI to 10 on master

**Review by arlakshm — APPROVED** [review]

**Comment by arlakshm** [comment]
@kenneth-arista, @ysmanman, @saksarav-nokia, @mlok-nokia  for viz...

**Comment by wenyiz2021** [comment]
hi @lguohan can you help to merge?

**Comment by yxieca** [comment]
@wenyiz2021  is this change still needed in 202311? if yes, you need to create separate PR.

**Comment by wenyiz2021** [comment]
@yxieca I don't think this is needed in 202311
@jimmyzhai could you confirm why you added the label for 202311?

**Pull request merged**